src/harness/BenchmarkDotNet.Extensions/ExclusionFilter.cs

```diff
@@ -0,0 +1,31 @@
+﻿using BenchmarkDotNet.Filters;
+using BenchmarkDotNet.Running;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace BenchmarkDotNet.Extensions
+{
+    class ExclusionFilter : IFilter
+    {
+        private string filter;
+
+        public ExclusionFilter(string _filter)
```

Comment:
the "inclusive" `--filter` allows for many "filters" (example: `--filter abc bcd cba`), perhaps this one could support it too (for consistency)?

Reply:
The only thing I am worried about here is that we are doing this separate from the actual arg parsing, and if we get into a situation where we are processing n number of things for the filter I am afraid of introducing bugs in other parts of the parsing. I think for now I will leave it as just one filter, but we can add this later, and pay the testing cost, if we have a scenario that needs it.